spec/function.dd

```diff
@@ -2952,6 +2960,227 @@ $(H3 $(LNAME2 system-functions, System Functions))
         $(P System functions are $(B not) covariant with trusted or safe functions.
         )
 
+        $(BEST_PRACTICE When in doubt, mark `extern (C)` and `extern (C++)` functions as
+        `@system` when their implementations are not in D, as the D compiler will be
+        unable to check them. Most of them are `@safe`, but will need to be manually
```

Comment:
> I don't like this sentence because a. saying something about 'most functions not implemented in D' is a bit dubious and b. `@safe` has no clear definition for implementations in other languages.

Yeah, that's Walter's. I agree, that the claim about "most" foreign functions having safe interfaces is dubious. And even if they do, they should marked `@trusted`, not `@safe`.

But it's a "best practice" (i.e. ultimately meaningless), and it correctly says: when in doubt, `@system`. So I feel like that's good enough for me.